File: UCA-4/Arrays/Swift/linearSearch.swift
import Swift

//Function For Linear Search
//Returns Index if Found Else -1
func linearSearch(arr:[Int],num : Int) -> Int {
    for (index,value) in arr.enumerated() {
        if (num == value){
            return index;
        }
    }
    return -1;
}

// MARK: MAIN PROGRAM

//Change Values If You want
var arr : [Int] = [1,2,3,4,5]
var numToFound = 5;
print("Number Found At \(linearSearch(arr:arr,num:numToFound))")

File: UCA-4/Arrays/Java/linearSearch.java
import java.util.*;

class test{

	static int linearSearch(int arr[],int numToFound){
		for (int i=0;i<arr.length;i++) {
			if(numToFound==arr[i])
				return i;
		}
		return -1;
	}

	public static void main(String[] args) {
		int size = Integer.parseInt(args[0]);
		int[] arr = new int[size];
		for (int i=0;i<size;i++) {
			arr[i] = Integer.parseInt(args[i+1]);
		}
		int numToFound = Integer.parseInt(args[size+1]);
		System.out.println("Number "+numToFound+" Found at "+linearSearch(arr,numToFound));
	}
}